src/jdk.httpserver/share/classes/com/sun/net/httpserver/Authenticator.java

```diff
@@ -51,12 +61,20 @@
 
         private int responseCode;
 
+        /**
+         * Creates a {@code Failure} instance with given response code.
+         *
+         * @param responseCode The response code to associate with this
+         *                     {@code Failure} instance
+         */
         public Failure (int responseCode) {
             this.responseCode = responseCode;
         }
 
         /**
          * returns the response code to send to the client
```

Comment:
Initial Capital and add a period please.

Reply:
I will include this change in the separate issue I created to track the additional formatting needed in these files -- https://bugs.openjdk.java.net/browse/JDK-8252822